<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddSongs</class>
 <widget class="QMainWindow" name="AddSongs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add songs to collection</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/oce.png</normaloff>icons/oce.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="global_layout">
      <item>
       <layout class="QHBoxLayout" name="inner_layout">
        <item>
         <widget class="QGroupBox" name="allsongs_groupbox">
          <property name="title">
           <string>All songs in your songs folder</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QScrollArea" name="allsongs_scrollarea">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="allsongs_scrollarea_contents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>442</width>
                <height>367</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QTreeWidget" name="allsongs_list">
                 <property name="selectionMode">
                  <enum>QAbstractItemView::ExtendedSelection</enum>
                 </property>
                 <attribute name="headerVisible">
                  <bool>false</bool>
                 </attribute>
                 <column>
                  <property name="text">
                   <string notr="true">1</string>
                  </property>
                 </column>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="allsongs_search_layout">
             <item>
              <widget class="QLineEdit" name="allsongs_search_field">
               <property name="placeholderText">
                <string>Filter songs...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="button_layout">
          <item>
           <widget class="QToolButton" name="add_mapset_button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Add all maps in the mapset of selected map.</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/add_mapset.png</normaloff>icons/add_mapset.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="add_beatmap_button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Add selected beatmap or mapset.</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/add_map.png</normaloff>icons/add_map.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="remove_beatmap_button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Remove selected beatmap or mapset.</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/remove_map.png</normaloff>icons/remove_map.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="remove_mapset_button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="toolTip">
             <string>Remove all maps in the mapset of selected map.</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>icons/remove_mapset.png</normaloff>icons/remove_mapset.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="addsongs_groupbox">
          <property name="title">
           <string>Songs to add to collection</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QScrollArea" name="addsongs_scrollarea">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="addsongs_scrollarea_contents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>442</width>
                <height>394</height>
               </rect>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QTreeWidget" name="addsongs_list">
                 <property name="selectionMode">
                  <enum>QAbstractItemView::ExtendedSelection</enum>
                 </property>
                 <attribute name="headerVisible">
                  <bool>false</bool>
                 </attribute>
                 <column>
                  <property name="text">
                   <string notr="true">1</string>
                  </property>
                 </column>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="confirmation_buttons">
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
